doc/manpages/man1/getzones.1.md

```diff
@@ -15,21 +15,67 @@ running **atalkd**(8).
 
 # Options
 
-**-m**
+**-g**
```

Comment:
Please don't forget to document the `-c` option as well. :)

Reply:
ooooops!  Fixed now.  Also noticed that the corresponding -m option wasn't documented in nbplkup's man page either, so added a note in there.